de.metas.acct.base/src/main/sql/postgresql/ddl/functions/ProfitAndLossReport.sql

```diff
@@ -0,0 +1,39 @@
+DROP FUNCTION IF EXISTS ProfitAndLossReport(IN p_startDate timestamp, IN p_endDate timestamp, IN p_compareWithPrevious1Year boolean, IN p_compareWithPrevious3Years boolean);
+DROP FUNCTION IF EXISTS ProfitAndLossReport(IN p_startDate timestamp, IN p_endDate timestamp);
+
+CREATE OR REPLACE FUNCTION ProfitAndLossReport(IN p_startDate timestamp, IN p_endDate timestamp)
+    RETURNS TABLE
+            (
+                name            text,
+                value           text,
+                isSummary       text,
+                three_years_ago numeric,
+                two_years_ago   numeric,
+                one_year_ago    numeric,
+                current_period  numeric
```

Comment:
better name the columns as:
* value => AccountValue
* name => AccountName
* IsSummary => take out
* three_years_ago => Balance_3yAgo
* two_years_ago => Balance_2yAgo
* one_year_ago => Balance_1yAgo
* current_period => Balance

Reply:
updated trl